### PR TITLE
Added support for installing Cake.Bakery.

### PR DESCRIPTION
**Pull request by patriksvensson**
This PR adds support for installing the Cake.Bakery package to the current work space.

### CODE REVIEW

File: src/bakery/cakeBakery.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+var AdmZip = require('adm-zip');
+import * as vscode from 'vscode';
+import * as path from 'path';
+import * as fs from 'fs';
+
+export class CakeBakery {
+
+    public getTargetPath(): string {
+        if (vscode.workspace.rootPath) {
+            return path.join(vscode.workspace.rootPath, "tools/Cake.Bakery/tools/Cake.Bakery.exe");
```
Comment (mholo65) [inline src/bakery/cakeBakery.ts]: We should reslly read this from `cake.config` as OmniSharp also does it. But that could be a separate PR.

Comment (patriksvensson) [reply src/bakery/cakeBakery.ts]: Ah, absolutely. I think however that the amount of people using the cake.config is pretty low, so for now we should be ok with it (until we do it properly).

File: package.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "displayName": "Cake",
   "publisher": "cake-build",
   "description": "Cake build script language support.",
-  "version": "0.8.0",
+  "version": "0.9.0-unstable0037",
```
Comment (patriksvensson) [inline package.json]: Umm. This got here by accident. Will remove.

**Comment by gep13** [comment]
Relates to #55

**Comment by gep13** [comment]
@patriksvensson your changes have been merged, thanks for your contribution 👍
